### PR TITLE
Cleanup manufacturing and friends

### PR DESCRIPTION
**Pull request by martukas**
Did the following:
* restructured manufacturing dir to reflect the pizza/enclosed dichotomy
* R&D stuff moved to manufacturing dir
* added and cleaned up pictures of India build, one visible from main page
* moved test lung stuff out to the QA directory
* cleaned up some ticket links and remaining references to pcbreathe (monorepo migration leftovers)
* fixed various other odd links

### CODE REVIEW

File: manufacturing/README.md [hunk 1]
```diff
@@ -1,13 +1,72 @@
...
-* [Beta Build Plan](beta-build-instructions)
-* [Production methods for custom components](assets/covent-june-2020-production-methods-for-custom-components.pdf)
+Broadly there are two types of ventilator builds we are currently testing with:
+- open table-top prototypes for testing components and various pneumatic configurations, kindly referred to as "pizza build"
+- "enclosed build", more like the final product; more expensive to make and slower to iterate on
 
-# Part Purchasing Guidelines
+| Pizza build  |  Enclosed build   |
+|:---------------------:|:--------------------:|
+![](pizza_build/graphics/3_pizzas_small.jpg) |![](enclosed_build/graphics/india1.jpeg) |
```
Comment (gracecanfield) [inline manufacturing/README.md:12]: Heads up - for some reason I'm not seeing images in previous within this table.

File: manufacturing/README.md [hunk 2]
```diff
@@ -1,13 +1,72 @@
...
+
+If you are a member of the team and intend to build your own prototype, please read the
+[purchasing guidelines](#part-purchasing-guidelines) first.
+
+## Sub-assemblies
+
+Components and sub-assemblies have firstly been tested and documented in the context of the pizza build. They should
+generally be similar for the enclosed build, but documentation might be lagging.
+
+For ech component, the goal is to document:
```
Comment (gracecanfield) [inline manufacturing/README.md:22]: "each"

File: manufacturing/enclosed_build/README.md [hunk 3]
```diff
@@ -1,23 +1,41 @@
-# Beta Build Plan
+# Enclosed build
+
+Preliminary enclosed build was produced by our partners, Foundry-M in Visakhapatnam, India.
+
+## Insides
+
+|     |     |
+|:---:|:---:|
+|![](graphics/insides.jpg) | ![](graphics/open_back.jpeg)  |
```
Comment (gracecanfield) [inline manufacturing/enclosed_build/README.md:9]: Not seeing images in preview FYI

File: manufacturing/enclosed_build/README.md [hunk 4]
```diff
@@ -1,23 +1,41 @@
...
+
+
+## With software
+
+|     |     |
+|:---:|:---:|
+|![](graphics/india1.jpeg) | ![](graphics/india2.jpeg) |
```
Comment (gracecanfield) [inline manufacturing/enclosed_build/README.md:16]: Not seeing images in preview FYI

File: manufacturing/README.md (hunk 1 above)
Comment (martukas) [reply manufacturing/README.md:12]: If you had pulled locally, you probably don't have LFS enabled. It's a new thing we have for binary files. Check repo wiki for updates.

File: manufacturing/README.md (hunk 2 above)
Comment (martukas) [reply manufacturing/README.md]: Got it, Thanks!

**Pull request merged**